api/VideoTrack.json

```diff
@@ -55,10 +55,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/VideoTrack/id",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
Oops, you only changed some of the entries to false in this file and AudioTrack.json.